### PR TITLE
docs: add version history

### PR DESCRIPTION
**Pull request by AdamEECS**

### CODE REVIEW

File: docs/en/14-reference/03-connector/06-rust.mdx [hunk 1]
```diff
@@ -27,9 +27,14 @@ The source code for the Rust connectors is located on [GitHub](https://github.co
...
 Websocket connections are supported on all platforms that can run Go.
 
-## Version support
-
-Please refer to [version support list](/reference/connector#version-support)
+## Version history
+
+| connector-rust version |  TDengine version |                   major features                    |
+| :----------------: | :--------------: | :--------------------------------------------------: |
+|       v0.8.8       |      3.0.5.0     | TMQ: get assignments and seek offset. |
```
Comment (gccgdb1234) [inline docs/en/14-reference/03-connector/06-rust.mdx]: TDengine version中修改一下，改成 >= 3.0.5.0 或者 3.0.5.0 or later

Comment (gccgdb1234) [inline docs/en/14-reference/03-connector/06-rust.mdx]: get assignments 是个内部用词。用户看不懂。改成： Get consuming progress and seek offset to consume

**Review by gccgdb1234 — APPROVED** [review]

**Comment by AdamEECS** [comment]
TD-24211

File: docs/en/14-reference/03-connector/06-rust.mdx (hunk 1 above)
Comment (AdamEECS) [reply docs/en/14-reference/03-connector/06-rust.mdx]: > TDengine version中修改一下，改成 >= 3.0.5.0 或者 3.0.5.0 or later

done

Comment (AdamEECS) [reply docs/en/14-reference/03-connector/06-rust.mdx]: > get assignments 是个内部用词。用户看不懂。改成： Get consuming progress and seek offset to consume

done

**Pull request merged**